web/static/js/bb-settings.js

```
@@ -632,6 +673,179 @@
+    );
+    if (!template) return;
+
+    try {
+      const schedules = await window.dataBinder.fetchData("/v1/schedulers", {
+        method: "GET",
+      });
+
+      const existing = schedulesList.querySelectorAll(
+        '.bb-job-card:not([data-settings-template="schedule"])'
+      );
+      existing.forEach((node) => node.remove());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)